s3/main.tf

```diff
@@ -2,8 +2,15 @@ locals {
   log_bucket_count = var.create_log_bucket ? 1 : 0
   log_bucket_name  = "${var.bucket_name}-logs"
 }
+
+resource "random_string" "bucket_suffix" {
+  length  = 8
+  special = false
+  numeric = false
```

Comment:
Why not numeric? Up the entropy!

Reply:
Fair point, I'll take that line out.

Reply:
Regarding excluding "special" characters, on the Terraform website it says "Include special characters in the result. These are `!@#$%&*()-_=+[]{}<>:?`"; does this mean that they don't count `,/"^\~` and the backtick character as special characters?

Reply:
> Bucket names can consist only of lowercase letters, numbers, dots (.), and hyphens (-). ([link](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html))

Reply:
Yes, but I was referring to the random string generator but I guess it would throw an error if the string generator generates a string with a disallowed character, right?